www/src/data/sidebars/doc-links.yaml

```diff
@@ -305,8 +305,8 @@
               link: /docs/seo/
             - title: Optimizing Site Performance with Guess.js
               link: /docs/optimizing-site-performance-with-guessjs/
-        - title: Localizing Your Site*
-          link: /docs/localization-i18n/
+        - title: Localization and Internationalization with Gatsby
+          link: /docs/gatsby-i18n/
```

Comment:
When changing the link it's best to also add a redirect to 
https://github.com/gatsbyjs/gatsby/blob/master/www/gatsby-node.js#L228

If you have no idea how to do that, just give a shout and I can do it for you. ❤️ 

(Great job on the rest of the PR!)

Reply:
It was a stub, so it didn't really seem necessary to me!

Reply:
I ended up changing the link back to the original (just changed the title), so no redirect needed!